webrtc-stats.html

```diff
@@ -1500,6 +1500,7 @@
              RTCIceCandidateType      candidateType;
              long                     priority;
              DOMString                url;
+             DOMString                turnUrlProtocol;
```

Comment:
Even though this protocol is included in the URL, I don't think "url" needs to be in the variable name. Plus, as Harald pointed out, it may not necessarily be in the URL. I'd suggest just "turnProtocol" or "relayProtocol".